packages/jsActions/nanoflow-actions-native/src/permissions/RequestGenericPermission.ts

```diff
@@ -0,0 +1,65 @@
+ * @returns {Promise.<"NanoflowCommons.Enum_PermissionStatus.unavailable"|"NanoflowCommons.Enum_PermissionStatus.denied"|"NanoflowCommons.Enum_PermissionStatus.limited"|"NanoflowCommons.Enum_PermissionStatus.granted"|"NanoflowCommons.Enum_PermissionStatus.blocked">}
+ */
+export async function GenericPermission(
```

Comment:
The function name should be `RequestGenericPermission`, right? I am taking reference from [RequestLocationPermission](https://github.com/mendix/widgets-resources/blob/master/packages/jsActions/nanoflow-actions-native/src/geolocation/RequestLocationPermission.ts#L15)

Reply:
done

Reply:
Verified & Resolved, thanks 👍